linera-service/src/linera.rs

```diff
@@ -848,6 +848,36 @@ enum ClientCommand {
         balance: Amount,
     },
 
+    /// Subscribes to a system channel.
+    Subscribe {
+        /// Chain id (must be one of our chains).
+        #[arg(long = "subscriber")]
+        subscriber: Option<ChainId>,
+
+        /// Chain id (must be one of our chains).
```

Comment:
Any owner should be fine but they need to process their inbox and create a block to accept the subscription (for now at least -- we have plans to remove this constraint)

Reply:
Thanks for reviewing it, can you elaborate bit more?

Reply:
I'm just saying that the comment "(must be one of our chains)" does not apply

Reply:
Ok